### PR TITLE
Overview header

### LINKED ISSUE #786: Implement header

### PR DESCRIPTION
**Pull request by y4izus**
Fix #786

### CODE REVIEW

File: components/ui/PageHeader.vue [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+import { Component } from 'vue-property-decorator'
+
+@Component
+export default class extends Vue {}
+</script>
+
+<style lang="scss" scoped>
+@import '~/assets/scss/blocks/copy.scss';
+@import '~carbon-components/scss/globals/scss/typography';
+@include carbon--theme($carbon--theme--white);
```
Comment (delapuente) [inline components/ui/PageHeader.vue:22]: If I don't recall wrong, I think this is the first time I see adding the `carbon--theme` mixing in a component. Why are we doing this?

File: pages/learn.vue [hunk 2]
```diff
@@ -1,16 +1,12 @@
...
-        <h1 class="the-learn-header__title">
-          Start your path towards<br>
-          learning
-          <span class="the-learn-header__relevant-word">
-            Qiskit
-          </span>
-        </h1>
-      </div>
-    </header>
+    <PageHeader>
```
Comment (delapuente) [inline pages/learn.vue:3]: Thank you for factoring out this!

File: components/ui/PageHeader.vue [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+      height: 28rem * 40 / 64;
+    }
+  }
+
+  &__title {
+    @include type-style('expressive-heading-05', true);
+    color: $white-text-01;
+  }
+
+  &__relevant-word {
```
Comment (delapuente) [inline components/ui/PageHeader.vue:49]: To preserve the separation of concerns, the header component should not style deep content. Also, we expect the "relevant word" to become a separated component where we implement the typewriter effect. I would prefer if we convert it in a component now (adding the typewriting effect in a follow-up PR) or if we convert it in its own BEM block, with its own identity.

File: components/ui/PageHeader.vue (hunk 1 above)
Comment (y4izus) [reply components/ui/PageHeader.vue]: Per offline conversations, it's a passing experiment. I will remove it

**Pull request merged**